### ISSUE TITLE
VNC Client & Server should connect to VNC repeater

### ISSUE THREAD

**Issue by crypticspawn** [TigerVNC/tigervnc#451]
I would like to connect tigervnc to a reflector/repeater (http://www.karlrunge.com/x11vnc/ultravnc_repeater.pl) looking at tightvnc most of my changes would be in the rfb folder, however, having a hard time figuring out how to attack it under tiger VNC.  I saw some parsing happening in vncviewer.cxx, I think the parsing since it happens in the viewer and the server so I have been looking for a cxx that does the parsing of the connection string, then all I would have to do is take out the reflector string and when I make an connection pass what I need to link it to the proper server.

**Comment by CendioOssman** [comment]
Yeah, most of it will probably be in common/rfb/ somewhere as that is where the protocol parsing happens. However there is some socket handling going on in vncviewer/ as well, so it might also need changes.

We would also like to see this extension documented in the @rfbproto project before merging.

**Comment by niterain** [comment]
CendioOssman is right.  There is Hostname.cxx, you will need to change, that is used everywhere, however, in some cases you won't be dealing with many changes in the rfb directory.  I changed ConnParams.h, and ConnParams.cxx.  The rest were in their respective directories.  The only problem I had problems with was getting the command line -connect to popup the AddNewClient Dialog, or pushing the connection string, just doesn't seem to work from the command line.  Windows 10 here, tigervnc 1.8.0.

**Comment by niterain** [comment]
I made my changes to a specific version, I am in the process of moving my changes to the trunk, so I should be issuing a pull request this week.

**Comment by q2dg** [comment]
Hello.
Any news on this?
Thanks!

**Comment by kintaro1981** [comment]
is latest version able to connect to a reflector/repeater?

**Comment by CendioOssman** [comment]
No, afraid not.

As a general rule, if an issue is still open then the feature has not yet been added.
